build/Versions.props

```diff
@@ -4,6 +4,13 @@
     <VersionBase Condition="'$(UseVisualStudioVersion)' != 'true'">2.8.0</VersionBase>
     <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
 
+    <!-- 
+      15.8 specific workaround for a change to BUILD_BUILDNUMBER format that added two more digits. 
```

Comment:
Tagging @dpoeschl specifically since this will require work to not take it when merging back to master.

Reply:
(although I guess it's written in a way that will be fine still...)

Reply:
Let's not merge it to master.